npm/ng-packs/packages/core/src/lib/directives/caps-lock.directive.ts

```diff
@@ -21,7 +21,7 @@ export class TrackCapsLockDirective {
     if (
       (s.toUpperCase() === s && s.toLowerCase() !== s && e.shiftKey) ||
       (s.toUpperCase() !== s && s.toLowerCase() === s && e.shiftKey) ||
-      e.getModifierState('CapsLock')
+      (e.getModifierState && e.getModifierState('CapsLock'))
```

Comment:
Can you try `!!e?.getModifierState('CapsLock')` condition ? It this works fine ?